notification-portlet-webapp/build.gradle

```diff
@@ -109,7 +109,7 @@ artifacts {
 
 task copyWebComponentsJs(type: Copy, dependsOn: ':notification-portlet-webcomponents:buildAllComponents' ) {
     from("${rootProject.projectDir}/notification-portlet-webcomponents") {
-        include "*/build/static/js/*.js"
+        include "*/build/static/js/**"
```

Comment:
Not working, apparently.

Reply:
Looks like `index.html` is getting copied over the javascript files